src/Orc.FileAssociation.Tests/FileAssociationsServiceFacts.cs

```diff
@@ -10,16 +10,18 @@ public class FileAssociationsServiceFacts
     {
         [Test]
         [TestCaseSource(nameof(Cases))]
-        public async Task AssociateFilesWithApplicationAsyncTestAsync(ApplicationInfo applicationInfo, List<String> extensions, string path)
+        public async Task AssociateFilesWithApplicationAsyncTestAsync(ApplicationInfo applicationInfo, List<string> classesSubKey, string subKey, string subKeyValue, string name)
```

Comment:
I think it would be better if you name parameters like `expectedClassesSubKey` etc